lib/analyzer/applications/php.ts

```diff
@@ -0,0 +1,99 @@
+import * as lockFileParser from "@snyk/composer-lockfile-parser";
+import { legacy } from "@snyk/dep-graph";
+import * as path from "path";
+import { DepGraphFact, TestedFilesFact } from "../../facts";
+
+import { DepTreeDep } from "../../types";
+import { AppDepsScanResultWithoutTarget, FilePathToContent } from "./types";
+
+interface ManifestLockPathPair {
+  manifest: string;
+  lock: string;
+}
+const PACKAGE_MANAGER_TYPE = "composer";
+
+export async function phpFilesToScannedProjects(
+  filePathToContent: FilePathToContent,
+): Promise<AppDepsScanResultWithoutTarget[]> {
+  const scanResults: AppDepsScanResultWithoutTarget[] = [];
+
+  const filePairs = findManifestLockPairsInSameDirectory(filePathToContent);
+
+  const shouldIncludeDevDependencies = false;
+
+  for (const pathPair of filePairs) {
+    const parserResult = await lockFileParser.buildDepTree(
+      filePathToContent[pathPair.lock],
+      filePathToContent[pathPair.manifest],
+      pathPair.manifest,
+      {},
+      shouldIncludeDevDependencies,
+    );
+
+    const depGraph = await legacy.depTreeToGraph(
```

Comment:
Do we have a type for this?

Reply:
Yes. It's `DepGraph` from `@snyk/dep-graph`

Reply:
Do you feel it's missing an explicit type declaration?